fdbserver/VersionedBTree.actor.cpp

```diff
@@ -1672,7 +1682,9 @@ struct RedwoodMetrics {
 			                                               { "", 0 } };
 
 		double elapsed = now() - startTime;
-
+		kvSizeWritten->writeToLog();
```

Comment:
I'd like the histogram log period to be separately configurable from the RedwoodMetrics trace event interval.  Also, logging the histograms in `getFields()` is a little strange, API-wise, because `getFields()` doesn't necessarily produce trace events.

Let's make another function called `logHistograms()` and have the metrics logger actor call it on a different cadence.  That knob can be `REDWOOD_HISTOGRAM_INTERVAL`, defaulting to 30, and we should rename `REDWOOD_LOGGING_INTERVAL` to `REDWOOD_METRICS_INTERVAL` for clarity.